viewer/packages/rendering/src/glsl/sector/sectorCoverage.frag

```diff
@@ -11,11 +11,11 @@ in lowp vec2 v_seed;
 in vec3 v_viewPosition;
 
 out vec4 outputColor;
-    
+
 const NodeAppearance dummyNodeAppearance = NodeAppearance(vec4(0.0), false, false, false);
 
 void main() {
-    if(v_visible != 1.0 || isClipped(dummyNodeAppearance, v_viewPosition)){
+    if((v_visible < 0.99999) || isClipped(dummyNodeAppearance, v_viewPosition)){
```

Comment:
Or perhaps a little more idiomatic floating point comparison?
```suggestion
    const epsilon = 0.0001;
    var isVisible = abs(v_visible - 1.0) < epsilon;
    if(!isVisible || isClipped(dummyNodeAppearance, v_viewPosition)){
```

Reply:
Ah, I just see this now.

Reply:
I commited an update (w small changes due to glsl syntax).